mqt/benchviewer/main.py

```diff
@@ -123,9 +142,28 @@ def get_num_benchmarks():
         return jsonify(data)
 
 
-def main():
-    app.run(debug=True)
+def start_server(
+    skip_question=False,
+    activate_logging=False,
+    target_location: str = "mqt/benchviewer/static/files/qasm_output",
+    debug_flag=False,
+):
+    init(
+        skip_question=skip_question,
+        activate_logging=activate_logging,
+        target_location=target_location,
+    )
+    print(
+        "Server is hosted at: " + "http://127.0.0.1:5000" + PREFIX + ".",
+        "To stop it, interrupt the process (e.g., via CTRL+C). \n",
+    )
+
+    if not activate_logging:
+        log = logging.getLogger("werkzeug")
```

Comment:
It was a copy-paste error from a code snippet. I implemented now a better way to avoid the default start-up message from flask by using:

`cli.show_server_banner = lambda *args: None`

Reply:
... update: unfortunately, to get rid of all loggings messages, the following lines must be re-added

```
        log = logging.getLogger("werkzeug")
        log.disabled = True
```

The reason for "werkzeug" is, that this is the default logger for basic request/response information.

Reply:
Alright. Then this should be fine. 👍🏻